lib/screens/filters_screen.dart

```diff
@@ -1,18 +1,88 @@
 import 'package:flutter/material.dart';
 import 'package:meals_app/widgets/main_drawer.dart';
 
-class FiltersScreen extends StatelessWidget {
+class FiltersScreen extends StatefulWidget {
   static const routeName = '/filters';
 
+  @override
+  _FiltersScreenState createState() => _FiltersScreenState();
+}
+
+class _FiltersScreenState extends State<FiltersScreen> {
+  var _glutenFree = false;
+  var _vegetarian = false;
+  var _vegan = false;
+  var _lactoseFree = false;
+
+  Widget _buildSwitchListTile(
+    String title,
+    String description,
+    var currentValue,
+    Function updateValue,
+  ) {
+    return SwitchListTile(
+      title: Text(title),
+      value: currentValue,
+      subtitle: Text(description),
+      onChanged: updateValue,
+    );
+  }
+
+  void changeGlutenFree(newValue) => setState(() => _glutenFree = newValue);
+
+  void changeLactoseFree(newValue) => setState(() => _lactoseFree = newValue);
+
+  void changeVegetarian(newValue) => setState(() => _vegetarian = newValue);
+
+  void changeVegan(newValue) => setState(() => _vegan = newValue);
```

Comment:
@MapleSyyrup this is what I'm looking for. Please apply this kind of practice on your succeeding pull requests. 😄